tree/ntuple/v7/doc/BinaryFormatSpecification.md

```diff
@@ -802,6 +802,27 @@ This section is a comprehensive list of the C++ types with RNTuple I/O support.
 Within the supported type system complex types can be freely composed,
 e.g. `std::vector<MyEvent>` or `std::vector<std::vector<float>>`.
 
+### Type Name Normalization
+
+Type names are stored according to the following normalization rules
+  - All whitespaces are removed.
+  - CV qualifiers (`const`, `volatile`) are removed.
+  - The `class`, `struct`, and `enum` keywords are removed.
+  - Type names are fully qualified by the namespace in which they are declared.
+  - The integer types `signed char`, `unsigned char`, and `[signed|unsigned](short|int|long[ long])` are replaced by
```

Comment:
What about `cont char*`?

Reply:
`char *` and by extension `const char *` are not supported by RNTuple. Users need to use `std::string`.